src/Elastic.Apm/DiagnosticSource/DiagnosticInitializer.cs

```diff
@@ -18,11 +24,24 @@ internal class DiagnosticInitializer : IObserver<DiagnosticListener>, IDisposabl
 
 		public void OnNext(DiagnosticListener value)
 		{
+			_logger.Trace()?.Log(nameof(OnNext) + " called with value.Name: {DiagnosticListenerName}", value.Name);
```

Comment:
Below is a comment and this will also log similarly for every single call. Not sure if this is really useful... maybe this one is fine...

Reply:
update: maybe it's not below, but above.. :)

Reply:
I agree, this log statement doesn't bring much value. Removed.